### PR TITLE
loc 0.3.2 (new formula)

### PR DESCRIPTION
**Pull request by kevinastone**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

Rust based Lines of Code counter

### CODE REVIEW

File: Formula/loc.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/loc"
+  end
+
+  test do
+    system bin/"loc", "--help"
```
Comment (bfontaine) [inline Formula/loc.rb]: Please write a test that does something more substantial than printing the help. You may want to create a dummy file and count its lines of code.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by kevinastone** [comment]
Added a sample file.

**Comment by kevinastone** [comment]
ping

**Review by bfontaine — APPROVED** [review]

**Comment by bfontaine** [comment]
@kevinastone Please don’t ping before 24-48 hours. I’ll have access to my Mac for the first time since Monday tonight; i.e. in 3-4 hours.

**Comment by kevinastone** [comment]
@bfontaine it was ~24 hours.  I'll be more patient, sorry.

**Comment by bfontaine** [comment]
…aaaaannnd merged! Thank you @kevinastone for your contribution to Homebrew! 🎉

**Pull request closed without merging**